src/Gaffer/Animation.cpp

```diff
@@ -1028,3 +1028,28 @@ ValuePlug::CachePolicy Animation::computeCachePolicy( const Gaffer::ValuePlug *o
 
 	return ComputeNode::computeCachePolicy( output );
 }
+
+bool Animation::equivalentValues( const double a, const double b )
+{
+	// see python PEP 485 : https://www.python.org/dev/peps/pep-0485
+
+	if( a == b )
+	{
+		return true;
+	}
+
+	if( std::isinf( a ) || std::isinf( b ) )
+	{
+		return false;
+	}
+
+	const double delta = std::abs( a - b );
+
+	static const double relative = 1E-9;
+	static const double absolute = std::numeric_limits< double >::epsilon();
```

Comment:
The PEP above says this :

> The default absolute tolerance value is set to zero because there is no value that is appropriate for the general case. It is impossible to know an appropriate value without knowing the likely values expected for a given use case. 

Perhaps another suggestion that the logic should be on the client side rather than hardcoded in here?